authentication/authentication-common/src/main/java/io/mosip/authentication/common/service/filter/InternalAuthenticationFilter.java

```diff
@@ -51,17 +51,17 @@ protected String extractBioData(String dataFieldValue) throws IdAuthenticationAp
 
 	@Override
 	protected boolean isSigningRequired() {
-		return env.getProperty("mosip.ida.internal.signing-required", Boolean.class, true);
+		return false;
 	}
 
 	@Override
 	protected boolean isSignatureVerificationRequired() {
-		return env.getProperty("mosip.ida.internal.signature-verification-required", Boolean.class, true);
+		return true;
 	}
 
 	@Override
 	protected boolean isTrustValidationRequired() {
-		return env.getProperty("mosip.ida.internal.trust-validation-required", Boolean.class, true);
+		return false;

```

Comment:
this should return 'true'

Reply:
@LoganathanSekar7627 as per story "For new internal API, signature validation is enabled and all other validations are disabled."

Reply:
@LoganathanSekar7627 in property file, it is disabled "mosip.ida.internal.trust-validation-required=false"

Reply:
@manojsp12 , It seems was mistakenly disabled, it should be enabled for new internal auth. Trust validation is also relates to the signature verification which needs to be enabled for new internal auth.

Reply:
@LoganathanSekar7627 Updated